app/assets/stylesheets/local/utilities.scss

```diff
@@ -0,0 +1,49 @@
+// utility classes used as overrides, therefore valid use of !important
```

Comment:
This is for Clive to decide, but this stuff makes me a bit uncomfortable. Do we really need to use !important for this? Aren't we just saying "give these margin settings to p.step-info"? If that's the case, it's simple enough to do with "proper" CSS, rather than chucking in !important.

Also, even if we do decide to chuck in some utility class hacks, can we only import the ones we are actually using? I think unused CSS classes (or any other code) just confuses things.